lib/convert.js

```diff
@@ -57,10 +58,21 @@ var sdk = require('postman-collection'),
 
       // set default options
       util.options = {
-        requestNameSource: (options.hasOwnProperty('requestNameSource') ? options.requestNameSource : 'fallback'),
-        schemaFaker: (options.hasOwnProperty('schemaFaker') ? options.schemaFaker : true),
-        indentCharacter: (options.hasOwnProperty('indentCharacter') ? options.indentCharacter : ' '),
-        collapseLongFolders: (options.hasOwnProperty('collapseLongFolders') ? options.collapseLongFolders : true)
+        requestNameSource: (options.hasOwnProperty('requestNameSource') ? options.requestNameSource :
+          definedOptions[0].default),
+        schemaFaker: (options.hasOwnProperty('schemaFaker') ? options.schemaFaker :
+          definedOptions[1].default),
+        indentCharacter: (options.hasOwnProperty('indentCharacter') ? options.indentCharacter :
```

Comment:
Indices cannot be hard-coded. Need to convert to a map and use lodash.merge to assign defaults